packages/components/src/components/Provider/Provider.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { ChakraProvider } from '@chakra-ui/react';
+import { ChakraProvider, ChakraProviderProps } from '@chakra-ui/react';
 
 export interface ProviderProps {}
```

Comment:
For some reason the GitHub suggestions interface is bugging out when I try to select multiple lines so I'll have to spread the suggestions across multiple comments 😬 

Since we're already using `interface` here, we can use `extends` to clean this up a bit.

```suggestion
export interface ProviderProps extends ChakraProviderProps {}
```